src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -1735,10 +1735,23 @@ BC30306: Array subscript expression missing.
             ]]>.Value
 
         ParseAndVerify(code, <errors>
-                                 <error id="30241"/>
+                                 <error id="37303"/>
```

Comment:
><error id="37303"/> [](start = 33, length = 19)

Please switch all affected tests to using AssertTheseDiagnostics helper, I see a bunch of tests using it in this file.